ui/browser/views/navbar/btn.jsx

```diff
@@ -12,21 +12,41 @@ specific language governing permissions and limitations under the License.
 
 import React, { PropTypes } from 'react';
 
+const HEIGHT = "20px";
+const WIDTH = "20px";
+
 /**
  * A button for the NavBar.
- * `icon` must be a valid font-awesome icon name
+ * `image` must be a valid CSS path to an image.
  */
-const Btn = ({ disabled, title, icon, onClick }) => (
-  <a href="#" className={disabled ? 'disabled' : ''} {...{ title, onClick }}>
-    <i className={`fa fa-${icon}`} />
+const Btn = ({ disabled, title, image, onClick }) => (
+  <a href="#"
+     style={{
+       backgroundImage: image ? `url(assets/${image})` : `url(${DEFAULT})`,
```

Comment:
I'm not a fan of the implications of sticking all of the styles into JS.  In this case I'm on board with passing in 'image' which sets inline style for background image instead of using icon class, but I'd expect the rest of the styles to live in CSS.  Specifically:

* For theming, how would that work?  Would all the various colors / font sizes be passed in as props to the component?  Seems easier to override the styles using CSS selectors.
* What about hover states?
* What about animations?

That said, I haven't used CSS in this way before so maybe I'm missing obvious answers and I'm open to be convinced.  But if it involves pulling in more libraries to handle these things, I wonder if it's adding enough value to justify it.  So it's not an r- but I want to discuss further about what the benefit is.